flower_shop/views/flower_in_products_menu.xml

```diff
@@ -0,0 +1,16 @@
+<odoo>
+    <data>
+        <!-- Define the window action -->
+        <record model="ir.actions.act_window" id="action_flower_product">
+            <field name="name">Flower Products</field>
+            <field name="res_model">product.product</field>
+            <field name="view_mode">kanban,tree,form,activity</field>
```

Comment:
Most of the time, you only ever need **tree** and **form** **modes**. If you specify **other** modes, be sure to **define the associated views** to have better control over the rendering.